DOC-2044 Fixing factor-of-ten error.

Colin-Fredericks opened the pull request:
JIRA ticket:
https://openedx.atlassian.net/browse/DOC-2044

Typo fix: 93,000,000 is 9.3\*10^7, not 9.3\*10^6.

edx-webhook commented:
Thanks for the pull request, @Colin-Fredericks! I've created [OSPR-626](https://openedx.atlassian.net/browse/OSPR-626) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams. 

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ('this must be merged by XX date', and why that is)
- partner information ('this is a course on edx.org')
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the Github pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

We can't start reviewing your pull request until you've submitted a [signed contributor agreement](http://open.edx.org/sites/default/files/wysiwyg/individual-contributor-agreement.pdf) or indicated your institutional affiliation and added yourself to the [AUTHORS](https://github.com/edx/edx-platform/blob/master/AUTHORS) file. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst) file for more information.

sarina commented:
@Colin-Fredericks thanks for this!

When you open up a PR, please link to the JIRA bug ticket if you are addressing a noted bug. That way, we can link back and forth. We also ask that you put the JIRA ticket number in your commit message.

Before I merge this, could you please add yourself to the AUTHORS list? Please do it in a separate commit; see https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst#step-1-sign-a-contribution-agreement for details.

sarina commented:
add to whitelist

Colin-Fredericks commented:
JIRA ticket:
https://openedx.atlassian.net/browse/DOC-2044

Added myself to the list and added the JIRA ticket number to the pull request comment. I already signed the contribution agreement a while ago, so we should be good on that front.

sarina commented:
@Colin-Fredericks yeah the bot was confused, because it is case sensitive.

Unfortunately it looks like your branch is behind master, which means there is a conflict with adding yourself to AUTHORS. You can rebase your branch, or you could remove the most recent commit and make a new pull request on top of the latest version of master and add yourself to authors there.

Colin-Fredericks commented:
Ok - I think it's up-to-date with edx's master now.

sarina commented:
@Colin-Fredericks I'm not sure what you did, but you added a third commit that contains a _bunch_ of work that's not yours.

We follow a rebasing workflow. Your branch should be rebased atop master (see instructions here: https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request) and your branch should only have two commits: one commit for the fix, and a second commit "Add myself to AUTHORS".

Please try rebasing and removing the last commit you added. If that is hard, I recommend just closing this pull request and making a new one that re-creates the commits. There is some fancy advance git magic you can do to fix the situation, but since these additions are so small it may not be worth it.

Colin-Fredericks commented:
Crap, I see what I did. Sorry. (Git is not my friend yet.)

Let's call this PR a loss. I'll just make another one tomorrow.
On Jun 3, 2015 5:58 PM, "Sarina Canelake" <notifications@github.com> wrote:

> @Colin-Fredericks <https://github.com/Colin-Fredericks> I'm not sure what
> you did, but you added a third commit that contains a *bunch* of work
> that's not yours.
>
> We follow a rebasing workflow. Your branch should be rebased atop master
> (see instructions here:
> https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request)
> and your branch should only have two commits: one commit for the fix, and a
> second commit "Add myself to AUTHORS".
>
> Please try rebasing and removing the last commit you added. If that is
> hard, I recommend just closing this pull request and making a new one that
> re-creates the commits. There is some fancy advance git magic you can do to
> fix the situation, but since these additions are so small it may not be
> worth it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/8337#issuecomment-108627227>.
>

sarina commented:
Yeah I getcha sometimes that is just easier. Feel free to close this one.
Or I will tomorrow.
On Jun 3, 2015 6:38 PM, "Colin-Fredericks" <notifications@github.com> wrote:

> Crap, I see what I did. Sorry. (Git is not my friend yet.)
>
> Let's call this PR a loss. I'll just make another one tomorrow.
> On Jun 3, 2015 5:58 PM, "Sarina Canelake" <notifications@github.com>
> wrote:
>
> > @Colin-Fredericks <https://github.com/Colin-Fredericks> I'm not sure
> what
> > you did, but you added a third commit that contains a *bunch* of work
> > that's not yours.
> >
> > We follow a rebasing workflow. Your branch should be rebased atop master
> > (see instructions here:
> > https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request)
> > and your branch should only have two commits: one commit for the fix,
> and a
> > second commit "Add myself to AUTHORS".
> >
> > Please try rebasing and removing the last commit you added. If that is
> > hard, I recommend just closing this pull request and making a new one
> that
> > re-creates the commits. There is some fancy advance git magic you can do
> to
> > fix the situation, but since these additions are so small it may not be
> > worth it.
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/edx/edx-platform/pull/8337#issuecomment-108627227>.
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/8337#issuecomment-108633755>.
>
